Invalid 'continue' outside loop on line 2
